docs/concepts/vega-chain.md

```diff
@@ -4,76 +4,75 @@ title: Vega Chain
+Vega uses Tendermint as a consensus layer to form a blockchain. The rest of the information here informs on how that blockchain and its relevant components is comprised. 
+
+Read more: [How Vega bridges to Ethereum](/docs/concepts/vega-chain/#bridges-used-for-staking)
 
 ## Delegated proof of stake
-Vega runs on a delegated proof of stake blockchain. Participants--validators and token-holders--use their VEGA tokens to nominate the validator nodes that run the network. Non-validator participants assign the voting rights of their VEGA tokens to endorse a validator's trustworthiness. 
+Vega runs on a delegated proof of stake blockchain. Participants -- validators and token-holders -- use their VEGA tokens to nominate validator nodes that run the network. Non-validator participants assign the voting rights of their VEGA tokens to endorse a validator's trustworthiness. 
+
+Read more: [Validator nodes](/docs/concepts/vega-chain#validator-nodes)
 
 **Participants who hold a balance of VEGA, the governance asset, can stake that asset on the network.** This is done by associating those tokens to a Vega key to use as stake, and then nominating one or more validators they trust to help secure the network. 
 
-Everyone participating in staking is rewarded for keeping the network running (and in the future can be penalised for not meeting the requirements of running the network). 
+Everyone participating in keeping the network secure, robust and reliable, including nominators, is **rewarded** for keeping the network running. Not meeting the requirements of running the network can lead to penalties, such as **rewards being withheld**.
 
-<!-- Penalties: 
-- Validators can lose their VEGA to the network if they don't meet the requirements or prove to be bad actors. The tokens are sent to the insurance pool. (link to insurance pool section when available) 
-- Nominators will lose future rewards
+Read more: [Rewards](/docs/concepts/vega-chain#rewards)
 
-### ***Further reading***
-Link to spec for staking and validator rewards when publicly available. (valpol)
--->
+Read more: [Penalties](/docs/concepts/vega-chain#penalties)
 
 ### VEGA token
-Vega uses the VEGA ERC20 token for governance, which includes nominating validators, and creating and voting on governance proposals.
+Vega uses the VEGA ERC20 token for governance, which includes nominating validators to run nodes, and creating and voting on governance proposals.
 
 A VEGA token (or fraction) can be either unassociated or associated with a Vega key:
 
 * **Unassociated**: The tokenholder is free to do what they want with the token, but cannot nominate a validator with it
 * **Associated**: The token is locked in the staking smart contract and can be used to nominate a validator. It must be unassociated to withdraw it
```

Comment:
suggest: Associated with a Vega key (using an ethereum smart contract) so that it can nominate a validator.

Reply:
I believe this is already covered by the line above, and I don't want to define associating by using the word associating :)